src/Test.UnitTests.Sarif.Driver/Sdk/FormatForVisualStudioTests.cs

```diff
@@ -13,22 +14,6 @@ namespace Microsoft.CodeAnalysis.Sarif.Driver
     // also exercise Region.FormatForVisualStudio and ResultKind.FormatForVisualStudio.
     public class FormatForVisualStudioTests
     {
-        private const string TestRuleId = "TST0001";
-        private const string TestMessageStringId = "testMessageStringId";
```

Comment:
Moved this data to a dedicated TestData class, which we should continue building out to hold all the test things. There are lots of important SARIF snippets to author (such as one in this PR in which we construct SARIF that explores the complete range of how to provide rule ids). There are analogs with logical locations, physical locations, etc. Need to make progress organizing these assets (and authoring comprehensive shared patterns). A WIP.